noah/src/main/scala/edu/uci/ics/cloudberry/noah/news/news.properties

```diff
@@ -0,0 +1,4 @@
+#Thu Jul 14 15:59:24 PDT 2016
```

Comment:
Try to maintain the `properties` file is too heavy. I'd prefer to just put these three arguments through the command line. Then we can create a script and let the script handle the external logics.